code-challenges.js

```diff
@@ -20,42 +20,77 @@ const fruit4 = "kiwi"
 
 // Pseudo code:
+// Create a function named "temp" that takes the boiling point and makes it an argument to return defined outcomes "is at boiling point", "is below boiling point" and "is above boiling point".  Our Boiling point = 212
 
 const temp1 = 42
 const temp2 = 350
 const temp3 = 212
 
+const temp = (boilingPoint) => {
+    if (boilingPoint === 212) {
+        return "is at boiling point"
+    } else if (boilingPoint < 212) {
+        return "is below boiling point"
+    } else if (boilingPoint > 212) {
+        return "is above boiling point"
+    }
+}
+console.log(temp(42))
+
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the variables provided below. Expected output: 10
 
 // Pseudo code:
+// First must join (concat) myNum1 and myNum2 to create new array.  Console log length of new array.
+
+// const myNum1 = [1, 2, 3, 4, 5]
+// const myNum2 = [6, 7, 8, 9, 6]
+// const myNum3 = myNum1.concat(myNum2)
+// console.log(myNum3.length) 
+// Answer is 10
 
-const myNumbers1 = [3, 7, 0, 36, -9]
-const myNumbers2 = [8, -7, 42, 9, 13]
 
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "2202 atleD"
 
 // Pseudo code:
-
-const currentCohort = "Delta 2022"
-
+// create function reverseWord, split all items in the string, reverse items in the string, and join items back in the string to create mirror of original string
+// function reverseWord (sentence) {
+//     return sentence.split(' ').map(function(word) {
+//       return word.split('').reverse().join('');
+//     }).join(' ');
+//   }
+  
+//   console.log(reverseWord("2022 Delta"));
+// Answer is 2202 atleD
 
 // --------------------4) Create a statement that will evaluate whether each number in an array is even or odd. Use the test variable provided below. Expected output: "odd" "even" "odd" "even" "odd" "even"
 
-// Pseudo code:
-
-const myArray = [13, 34, 5, 10, 27, 42]
+// Pseudo code:create array with index less than 7.
 
+// const myArray = [13, 34, 5, 10, 27, 42]
+// for(let i = 1; i < 7; i++){
+//     if(i % 2 !== 0){
+//         console.log("ODD")
+//     }else{
+//         console.log("EVEN")
+//     }
+// }
+// Answer is ODD EVEN ODD EVEN ODD EVEN
 
 // --------------------5) Create a function that takes in a set of any two numbers and subtracts the smaller number from the larger number using the sets of test variables provided below. Expected output: 42, 3
 
-// Pseudo code:
+// Pseudo code: Console log two numbers assigned to a const that subtracts the smaller variable from the larger variable.
```

Comment:
Make sure to pay attention to the prompt.  You are being asked to _create a function_ on this problem.